server/data/welcomeClient.test.ts

```diff
@@ -84,7 +84,7 @@ describe('welcomeClient', () => {
     })
     it('should return data from api', async () => {
       fakeWelcomeApi
-        .get(`/prisons/${activeCaseLoadId}/recent-arrivals?fromDate=${fromDate}&toDate=${toDate}&pageSize=50&page=0`)
+        .get(`/prisons/${activeCaseLoadId}/recent-arrivals?fromDate=${fromDate}&toDate=${toDate}`)
```

Comment:
Removed pageSize and page params as no longer needed. These are now defaulted to values in the back end.